### PR TITLE
Split out loadbalancer

### PR DESCRIPTION
**Pull request by volllly**

### CODE REVIEW

File: ParameterReference.md [hunk 1]
```diff
@@ -15,10 +15,7 @@ The following settings are valid:
...
-| proxy      | no                                     |                   null                    |                                                                NO_PROXY value according to https://docs.docker.com/network/proxy/#set-the-environment-variables-manually                                                                 |
-| proxy      | ftp                                    |                   null                    |                                                                FTP_PROXY value according to https://docs.docker.com/network/proxy/#set-the-environment-variables-manually                                                                |
+| proxy      |                                        |                   null                    |                                      Proxy string according to https://docs.fiskaltrust.cloud/docs/posdealers/technical-operations/middleware/network-requirements#setting-the-proxy-configuration                                       |
```
Comment (TSchmiedlechner) [inline ParameterReference.md:18]:
```suggestion
| proxy      |                                        |                   null                    |                                      Proxy string according to https://link.fiskaltrust.cloud/rollout/proxy                                       |
```

File: ParameterReference.md [hunk 2]
```diff
@@ -68,18 +61,15 @@ The following settings are valid:
...
-| scope        | singleNamespace |     true      |                                      If you want Emissary-ingress to only work outside of ByoDC, set to false                                       |
-| nodeSelector |                 |      {}       | Sets the `nodeSelector` property of the emissary ingress PODs https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector |
+| Subsection | Parameter | Default Value |                                   Description                                         |
+| :--------- | :-------- | :-----------: | :-----------------------------------------------------------------------------------: |
+| use        |           |     true      |                     Deploys the emissary ingress custom resources                     |
+| tls        |           |     false     | If you want Emissary-ingress to use tls (cert configured in the loadbalancer section) |
+| hostname   |           |      *        |       If you want Emissary-ingress to only work outside of ByoDC, set to false        |
```
Comment (TSchmiedlechner) [inline ParameterReference.md:75]: Is this the correct description for `hostname`?

File: QuickStart.md [hunk 3]
```diff
@@ -17,13 +17,17 @@ kubectl create namespace bring-your-own-datacenter
 
 ![](images/ByoDC-Quickstart-1-Namespace.png)
 
-## 3.) Add Helm Repo
+## 3.) Install Emissary Ingress
+
+Install Emissary Ingress v3.x using the [official installation instructions](https://www.getambassador.io/docs/emissary/latest/topics/install/helm).
```
Comment (TSchmiedlechner) [inline QuickStart.md:22]: Maybe we can pin the link to 3.x? E.g. `https://www.getambassador.io/docs/emissary/3.5/topics/install/helm`

File: how-to-multiple-installations.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Installing ByoDC multiple times on the same cluster
+
+To do this you will also need to install Imissary Ingress multiple times directly into each of the byodc namespaces.
```
Comment (TSchmiedlechner) [inline how-to-multiple-installations.md:3]:
```suggestion
To do this you will also need to install Emissary Ingress multiple times directly into each of the ByoDC namespaces.
```

**Review by TSchmiedlechner — APPROVED** [review]

File: QuickStart.md (hunk 3 above)
Comment (volllly) [reply QuickStart.md]: good idea 🙏

**Pull request merged**